ipn/local.go

```diff
@@ -342,9 +344,12 @@ func (b *LocalBackend) runPoller() {
 }
 
 func (b *LocalBackend) send(n Notify) {
-	if b.notify != nil {
+	b.mu.Lock()
+	notify := b.notify
+	b.mu.Unlock()
+	if notify != nil {
```

Comment:
I like Nigel Tao's policy of making a lock/unlock block its own paragraph.

Reply:
Done.